src/urlForImage.ts

```diff
@@ -166,20 +166,20 @@ function fit(
     }
 
     cropRect = {
-      left: Math.round(left),
-      top: Math.round(top),
-      width: Math.round(width),
-      height: Math.round(height),
+      left: Math.max(0, left),
+      top: Math.max(0, top),
```

Comment:
Should these `max` instead be applied when we define `left` and `top`? E.g:

```typescript
let left = Math.max(0, Math.round(hotspotXCenter - width / 2))
````